src/position.ts

```diff
@@ -847,6 +815,20 @@ export class UserPosition extends PositionWithRunner {
     return createEmptyPermitSignature();
   }
 
+  private async sendManagePositionTransaction<A extends Array<unknown>, R, S extends Exclude<StateMutability, 'view'>>(
+    method: TypedContractMethod<A, R, S>,
+    gasLimitMultiplier: Decimal,
+    args: { [I in keyof A]-?: A[I] },
+  ): Promise<ContractTransactionResponse> {
+    if (!gasLimitMultiplier.equals(Decimal.ONE)) {
```

Comment:
I think we should run gas estimate every time, because it allows us to detect contract errors during estimate, and prevent user from starting tx that will fail (waste gas).